cedar-policy-cli/build.rs

```diff
@@ -0,0 +1,25 @@
+fn main() {
+    generate_schemas();
```

Comment:
can we guard this on the `protobufs` feature, so that we don't need it if the feature is disabled?